### PR TITLE
Update README with some details about quick start and sample app

### PR DESCRIPTION
**Pull request by bkielbasa**
Do you think there's more info needed in the readme?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,80 @@
 
 [![GoDoc](https://godoc.org/github.com/go-bdd/gobdd?status.svg)](https://godoc.org/github.com/go-bdd/gobdd) [![Coverage Status](https://coveralls.io/repos/github/go-bdd/gobdd/badge.svg?branch=master)](https://coveralls.io/github/go-bdd/gobdd?branch=master)
 
-This is a BDD testing framework. Uses gherkin for the test's syntax. Project is under active development.
+This is a BDD testing framework. Uses gherkin for the test's syntax. From version 1.0, the API is stable.
 
-More detailed documentation can be found on the docs page: https://go-bdd.github.io/gobdd/
+## Why did I make the library?
+There is godog library for BDD tests in Go. I found this library useful but it run as an external application which compiles our code. It has a several disadvantages:
```
Comment (mirogta) [inline README.md:8]: typo: "it runs"

Comment (mirogta) [reply README.md:8]: "it has several disadvantages"

File: README.md [hunk 2]
```diff
@@ -2,17 +2,80 @@
...
+
+func TestScenarios(t *testing.T) {
+	suite := NewSuite(t)
+	suite.AddStep(`I add (\d+) and (\d+)`, add)
+	suite.AddStep(`I the result should equal (\d+)`, check)
+	suite.Run()
+}
+```
+
+Inside `features` folder create your scenarios. Here is an example:
```
Comment (mirogta) [inline README.md:55]: Would it make sense to move this section related to features above the section related to test steps? I usually write (or already have) feature files and only then go on implementing the steps.

Comment (bkielbasa) [reply README.md:55]: good point!

**Pull request merged**